Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -177,7 +177,7 @@
             return pair.value;
     }
 
-    ASSERT(m_webProcessProxy->hasConnection() && &connection == &m_webProcessProxy->connection());
+    RELEASE_ASSERT(m_webProcessProxy->hasConnection() && &connection == &m_webProcessProxy->connection());
```

Comment:
This can use the AuxiliaryProcessProxy::hasConnection that takes a connection parameter to simplify.